src/Sulu/Bundle/MediaBundle/Resources/config/services.xml

```diff
@@ -310,6 +310,7 @@
             <argument type="service" id="sulu_media.reference_store.media"/>
             <argument type="service" id="sulu_core.webspace.request_analyzer"/>
             <argument>%sulu_security.permissions%</argument>
+            <argument type="service" id="sulu_admin.property_metadata_min_max_value_resolver" on-invalid="null"/>
```

Comment:
Is the `on-invalid` really necessary? I think the service will always be there, won't it?

Reply:
Explained here https://github.com/sulu/sulu/pull/5666#discussion_r535240358